### PR TITLE
feat(kubernetes,gcp,aws): #20

### PR DESCRIPTION
**Pull request by ivankudibal**
basic k8s and gcp templates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,14 @@ Start by reading [how to use a template](docs/use_a_template.md), then check eac
 
 | Template             | Description | Author |
 |----------------------|-------------|:------:|
+| [AWS Cloudwatch Monitoring](aws_cloudwatch/) | Monitor AWS EC2 and ELB in your dashboard | [bonitoo.io](.) |
```
Comment (sanderson) [inline README.md:17]: Monitor AWS EC2 and ELB ~in your dashboard~

File: aws_cloudwatch/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# AWS Cloudwatch Monitoring Template
+
+Provided by: [bonitoo.io](.)
+
+**Display data from AWS EC2 andn ELB using the AWS Cloudwatch Service.**
```
Comment (sanderson) [inline aws_cloudwatch/README.md:5]: ~Cloudwatch~ -> CloudWatch

File: aws_cloudwatch/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+1. The AWS Cloudwatch Network Load Balancers monitoring dashboard displays data from `cloudwatch_aws_network_elb` measurement.
```
Comment (sanderson) [inline aws_cloudwatch/README.md:7]: The AWS ~Cloudwatch~ **CloudWatch** Network Load Balancers monitoring dashboard displays data from **the** `cloudwatch_aws_network_elb` measurement.

File: aws_cloudwatch/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+2. The AWS Cloudwatch Instance Monitoring dashboard displays data from the `cloudwatch_aws_ec2` measurement.
```
Comment (sanderson) [inline aws_cloudwatch/README.md:8]: The AWS ~Cloudwatch~ **CloudWatch** Instance Monitoring dashboard displays data from the `cloudwatch_aws_ec2` measurement.

File: aws_cloudwatch/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+Dashboard examples
```
Comment (sanderson) [inline aws_cloudwatch/README.md:10]: Make this an h5 or h6

File: aws_cloudwatch/README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+![AWS Cloudwatch Instance Monitoring](img/aws-cloudwatch-instance-monitoring.png)
+![AWS Cloudwatch NLB Monitoring](img/aws-cloudwatch-nlb-monitoring.png)
+
+## Included Resources
+
+The solution composes of the following resources:
```
Comment (sanderson) [inline aws_cloudwatch/README.md:17]: This template includes the following:

File: aws_cloudwatch/README.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+- 2 Labels: `inputs.cloudwatch`, `AWS`
+- 2 Dashboards: `AWS Cloudwatch Instance Monitoring`, `AWS Cloudwatch NLB Monitoring`
+- 1 Variable: `bucket`
+
+## Setup Instructions
+
+Load the dashboards and use the [Telegraf Cloudwatch plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/cloudwatch) into your
```
Comment (sanderson) [inline aws_cloudwatch/README.md:25]: Include the [Telegraf Cloudwatch plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/cloudwatch) in your Telegraf configuration and start Telegraf.

File: aws_cloudwatch/README.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+environment.
+
+Set the `v.bucket` variable in the dashboards or settings to let dashboards know
+what bucket holds the data.
```
Comment (sanderson) [inline aws_cloudwatch/README.md:29]: Visit the dashboard and use the `v.bucket` variable to select which bucket the data is stored in.

File: gcp_monitoring/README.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+**Display data from Google Cloud Monitoring -- formerly Stackdriver -- using the
+Cloud Monitoring API v3.**
+
+1. MySQL database hosted in Google Cloud SQL
+2. The Compute dashboard provides deeper insight into the GCP compute services
+   provided by GCP Stackdriver.
+3. HTTPS LoadBalancing dashboard monitors network load balancing in the cluster.
+
+Dashboard examples
```
Comment (sanderson) [inline gcp_monitoring/README.md:13]: Make this an h5 or h6

File: gcp_monitoring/README.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+![GCP Monitoring Cloud SQL](img/gcp-monitoring-cloudsql.png)
+![GCP Monitoring Compute](img/gcp-monitoring-compute.png)
+![GCP Monitoring LoadBalancing](img/gcp-monitoring-loadbalancing.png)
+
+## Included Resources
+
+The solution composes of the following resources:
```
Comment (sanderson) [inline gcp_monitoring/README.md:21]: This template includes the following:

File: gcp_monitoring/README.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+Provided by: [bonitoo.io](.)
+
+**Display data from Google Cloud Monitoring -- formerly Stackdriver -- using the
+Cloud Monitoring API v3.**
+
+1. MySQL database hosted in Google Cloud SQL
+2. The Compute dashboard provides deeper insight into the GCP compute services
+   provided by GCP Stackdriver.
+3. HTTPS LoadBalancing dashboard monitors network load balancing in the cluster.
```
Comment (sanderson) [inline gcp_monitoring/README.md:11]: Are these things included in the template? Requirements? etc. Need some context for what this list is.

File: gcp_monitoring/README.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+The solution composes of the following resources:
+
+- 2 Labels: `inputs.stackdriver`, `GCP`
+- 3 Dashboards: `CloudSQL`, `Compute` and `HTTPS LoadBalancing`
+- 1 Variable: `bucket`
+
+## Setup Instructions
+
+Load the dashboards and use the Telegraf Stackdriver plugin into your
+environment.
```
Comment (sanderson) [inline gcp_monitoring/README.md:30]: - For setup instructions, should we provide an example command for installing the template?
- Need a link to the Stackdriver plugin
- Include the [Telegraf Stackdriver plugin](#) in your Telegraf configuration and start Telegraf.

File: gcp_monitoring/README.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+
+Set the `v.bucket` variable in the dashboards or settings to let dashboards know
+what bucket holds the data.
```
Comment (sanderson) [inline gcp_monitoring/README.md:33]: Visit the dashboard and use the v.bucket variable to select which bucket the data is stored in.

File: gcp_monitoring/README.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+
+
+## Customizations
+
+n/a
+
+## Contact
+
+Provide a way for users to get in touch with you if they have questions or need help using your template. What information you give is up to you, but we encourage providing those below.
```
Comment (sanderson) [inline gcp_monitoring/README.md:42]: Remove this paragraph.

File: k8s/README.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
+# Kubernetes Dashboards Template
+
+Provided by: [bonitoo.io](.)
+
+**This template provides 2 basic Kubernetes dashboards:
+`Kubernetes Node Metrics` and `Kubernetes Inventory`. The K8S infrastrucure
+supports Google Cloud Platform, AWS as well as on-premise K8S environments.**
+
+
+Screenshots
```
Comment (sanderson) [inline k8s/README.md:10]: - Dashboard examples
- Make this an h5 or h6

File: k8s/README.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
+# Kubernetes Dashboards Template
+
+Provided by: [bonitoo.io](.)
+
+**This template provides 2 basic Kubernetes dashboards:
+`Kubernetes Node Metrics` and `Kubernetes Inventory`. The K8S infrastrucure
+supports Google Cloud Platform, AWS as well as on-premise K8S environments.**
```
Comment (sanderson) [inline k8s/README.md:7]: supports Google Cloud Platform, AWS, ~as well as~ **and** on-premise K8S environments.

File: k8s/README.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+
+Screenshots
+
+![Screenshot](img/k8s-nodemetrics-dashboard.png)
+![Screenshot](img/k8s-inventory-dashboard.png)
+
+
+## Included Resources
+
+The solution composes of the following resources:
```
Comment (sanderson) [inline k8s/README.md:18]: This template includes the following:

File: k8s/README.md [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+## Setup Instructions
+
+Set up and install Telegraf `kubernetes` and `kube_inventory` plugins into your
+Telegraf configmap.
+
+* https://github.com/influxdata/telegraf/tree/master/plugins/inputs/kubernetes
+* https://github.com/influxdata/telegraf/tree/master/plugins/inputs/kube
+
+Set the `v.bucket` variable in the dashboards or settings to let dashboards know
+what bucket holds the data.
```
Comment (sanderson) [inline k8s/README.md:33]: Visit the dashboard and use the `v.bucket` variable to select which bucket the data is stored in.

File: gcp_monitoring/README.md (hunk 12 above)
Comment (ivankudibal) [reply gcp_monitoring/README.md:32]: I've added example of command..

File: gcp_monitoring/README.md (hunk 11 above)
Comment (ivankudibal) [reply gcp_monitoring/README.md]: Three dashboards

**Pull request merged**